pkgs/applications/networking/browsers/chromium/default.nix

```diff
@@ -14,12 +14,13 @@
 , commandLineArgs ? ""
 }:
 
+assert stdenv.cc.isClang -> (stdenv == llvmPackages.stdenv);
```

Comment:
It would be nice to find a way to get ```llvmPackages``` from Clang's ```stdenv``` to simplify getting ```llvmPackages.lld``` or ```${llvmPackages.llvm}/bin/llvm-ar``` into derivation code without this hack of passing separate ```llvmPackages``` and checking it